libs/net/entities/TonePool.cs

```diff
@@ -25,15 +25,15 @@ public class TonePool : BaseType<int>
     #region Constructors
     protected TonePool() { }
 
-    public TonePool(string name, User owner) : base(name)
+    public TonePool(string name, int ownerId) : base(name)
```

Comment:
Undo your changes to this file as nothing changed.

Reply:
Comment added for the Elasticsearch case, thanks